lib/periodicChecks.js

```diff
@@ -174,7 +175,58 @@ const ensureAllIssuesHaveProjects = async (context) => {
   });
 };
 
+/**
+ * This function comments on and tags pull requests in which the last commit was created 2 days ago
+ * @param {import('probot').Context} context
+ */
+const checkAndTagPRsWithOldBuilds = async (context) => {
+  const OLD_BUILD_LABEL = "PR: don't merge - STALE BUILD";
+  console.log({MIN_BUILD_DATE: utilityModule.MIN_BUILD_DATE});
+
+  const allOpenPullRequests = await getAllOpenPullRequests(context);
+
+  allOpenPullRequests.forEach(async (pullRequest) => {
+    const containsOldBuildLabel = pullRequest.labels.some(
+      label => label.name === OLD_BUILD_LABEL
+    );
+    // If label has already been added, do nothing.
+    if (!containsOldBuildLabel) {
+    // Get last commit for pull request.
+    const lastCommitSha = pullRequest.head.sha;
+
+    const pRCommits = await context.github.pulls.listCommits(
+      context.repo({ pull_number: pullRequest.number })
+    );
+
+    const lastCommit = pRCommits.data.find(
+      (commit) => commit.sha === lastCommitSha
+    );
+    const lastCommitDate = new Date(lastCommit.commit.author.date);
+      // Commit is older than 2 days
+      if (utilityModule.MIN_BUILD_DATE > lastCommitDate) {
+        const labelParams = context.repo({
+          issue_number: pullRequest.number,
+          labels: [OLD_BUILD_LABEL],
+        });
+        context.github.issues.addLabels(labelParams);
+
+        // Ping PR author.
+        const commentParams = context.repo({
+          issue_number: pullRequest.number,
+          body:
+            'Hi @' + pullRequest.user.login + ', the build of this PR is ' +
+            'stale and this could result in tests failing in develop. Please' +
+            ' update this pull request with the latest changes from develop,' +
+            ' Thanks.',
+        });
```

Comment:
```suggestion
            'Hi @' + pullRequest.user.login + ', the build of this PR is ' +
            'stale and this could result in tests failing in develop. Please ' +
            'update this pull request with the latest changes from develop, ' +
            'Thanks.',
        });
```

Reply:
Adding these will make those lines 81 characters long.

Reply:
Included in a different review comment.